lib/nimble_template/templates/template.ex

```diff
@@ -31,82 +30,35 @@ defmodule NimbleTemplate.Templates.Template do
 
   defp post_apply!(%Project{mix_project?: true} = project) do
     order_dependencies!()
-    fetch_and_install_elixir_dependencies()
+    install_elixir_dependencies()
     suppress_credo_warnings_for_base_project(project)
     format_codebase()
   end
 
   defp post_apply!(%Project{api_project?: true} = project) do
     order_dependencies!()
-    fetch_and_install_elixir_dependencies()
+    install_elixir_dependencies()
     suppress_credo_warnings_for_phoenix_api_project(project)
     format_codebase()
   end
 
   defp post_apply!(%Project{web_project?: true} = project) do
     order_dependencies!()
-    fetch_and_install_elixir_dependencies()
-    fetch_and_install_node_dependencies()
+    install_elixir_dependencies()
+    install_node_dependencies()
     suppress_credo_warnings_for_phoenix_project(project)
     format_codebase()
   end
 
-  defp fetch_and_install_elixir_dependencies() do
+  defp install_elixir_dependencies() do
     Mix.shell().cmd("MIX_ENV=dev mix do deps.get, deps.compile")
     Mix.shell().cmd("MIX_ENV=test mix do deps.get, deps.compile")
   end
 
-  defp fetch_and_install_node_dependencies() do
+  defp install_node_dependencies() do
```

Comment:
It should reformat to `,do: ...` block and the `format_codebase` function. What do you think?

Reply:
Nice catch, updated in https://github.com/nimblehq/elixir-templates/pull/308/commits/ce27c66a70f9c83889b6fe91e171cd5a03d244e4